src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -1107,6 +1107,36 @@ table.matrix th.group-all-groups-cell {
     select, button {
         margin-left: 5px;
     }
+
+    .error {
+        display: none;
+    }
+
+    .rnaseq-search-autocomplete {
```

Comment:
This is not really the BEM naming style. We enforce the format but not the principle so technically it is okay as per for code review.

Reply:
oh my bad, I will push a change